net/FlatBuffers/TablePos.cs

```diff
@@ -1,50 +1,37 @@
-/*
- * Copyright 2014 Google Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
-namespace FlatBuffers
+namespace FlatBuffers 
 {
-    /// <summary>
-    /// All tables in the generated code derive from this class, and add their own accessors.
-    /// </summary>
-    public abstract class Table
+    public struct TablePos 
     {
-        protected int bb_pos;
-        protected ByteBuffer bb;
+        public TablePos(int bb_pos, ByteBuffer bb) 
+        {
+            this.bb_pos = bb_pos;
+            this.bb = bb;
+        }
 
-        public ByteBuffer ByteBuffer { get { return bb; } }
+        public readonly int bb_pos;
+        public readonly ByteBuffer bb;
 
         // Look up a field in the vtable, return an offset into the object, or 0 if the field is not
         // present.
-        protected int __offset(int vtableOffset)
+        public int __offset(int vtableOffset)
```

Comment:
Same. Public methods such as this really should conform to the C# style guide (Offset() or similar).